core/src/test/java/org/wildfly/extras/sunstone/tests/ec2/EC2SubnetTest.java

```diff
@@ -0,0 +1,59 @@
+package org.wildfly.extras.sunstone.tests.ec2;
+
+import com.google.common.base.Strings;
+import org.jclouds.aws.ec2.AWSEC2Api;
+import org.jclouds.aws.ec2.domain.AWSRunningInstance;
+import org.jclouds.ec2.domain.Reservation;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.wildfly.extras.sunstone.api.CloudProperties;
+import org.wildfly.extras.sunstone.api.CloudProvider;
+import org.wildfly.extras.sunstone.api.impl.Config;
+import org.wildfly.extras.sunstone.api.impl.ec2.EC2CloudProvider;
+import org.wildfly.extras.sunstone.api.impl.ec2.EC2Node;
+
+import java.util.Set;
+
+public class EC2SubnetTest {
```

Comment:
Add some documentation here, please. What it does and what's necessary to configure to run the tests?